### PR TITLE
HPCC-16767 Add HU translations

### PR DESCRIPTION
**Pull request by miguelvazq**
Add Hungarian translations.

Signed-off by: Miguel Vazquez <miguel.vazquez@lexisnexis.com>

### CODE REVIEW

**Comment by miguelvazq** [comment]
@GordonSmith Please review.

**Comment by GordonSmith** [comment]
@AttilaVamos Are those two trailing quotes correct?

File: esp/src/eclwatch/nls/hu/hpcc.js [hunk 1]
```diff
@@ -371,17 +439,23 @@ define(
...
     PleaseEnterANumber: "Adjon meg egy számot",
+    PleasePickADefinition: "Válasszon egyet a definíciók közül",
     PleaseSelectAUserOrGroup: "A fájlnévvel együtt adjon meg egy felhasználó vagy csoport azonosítót",
+    PleaseSelectADynamicESDLService: "Válasszon egyet a dinamikus ESDL szolgáltatások közül.",
+    PleaseSelectATopologyItem: "Válasszon egy célt, szolgáltatást vagy gépet",
     Plugins: "Betölthető modulok",
     Port: "Port",
     Prefix: "Előtag",
     PrefixPlaceholder: "fájlnév{:hossz}, fájlméret{:[B|L][1-8]}",
+    Preflight: "Előkészítés&rsquo;",
```
Comment (AttilaVamos) [inline esp/src/eclwatch/nls/hu/hpcc.js]: No, it is a typo

File: esp/src/eclwatch/nls/hu/hpcc.js [hunk 2]
```diff
@@ -465,18 +546,22 @@ define(
...
+    ShowProcessesUsingFilter: "Feldolgozások  megjelenítése szűréssel",
     ShowSVG: "SVG megjelenítés",
     Size: "Méret",
     Skew: "Asszimmetrikus (Skew)",
+    SkewNegative: "Negatív elcsúszás",
     SLA: "SLA",
     Slaves: "Kiszolgáló",
     SlaveLogs: "Slave logs",
     SlaveNumber: "Slave-ek száma",
+    SkewPositive: "Pozitív elcsúszás&rsquo;",
```
Comment (AttilaVamos) [inline esp/src/eclwatch/nls/hu/hpcc.js]: Same typo

**Comment by AttilaVamos** [comment]
@miguelvazq Both of them are typo. Please remove.

**Comment by richardkchapman** [comment]
Closing as replaced by #9437

**Pull request closed without merging**